lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -85,7 +87,8 @@
                           final ClasspathProvider classpath,
                           final YarnProxyUser yarnProxyUser,
                           final SecurityTokenProvider tokenProvider,
-                          final YarnSubmissionParametersFileGenerator jobSubmissionParametersGenerator) {
+                          final YarnSubmissionParametersFileGenerator jobSubmissionParametersGenerator,
+                          @Parameter(NodeLabelExpression.class) final String nodeLabelExpression) {
```

Comment:
Is the label for request evaluator only? If yes, why add it to Job submission? It should be in Evaluator request only.